src/daft-scan/src/glob.rs

```diff
@@ -85,6 +85,17 @@ fn run_glob(
     Ok(Box::new(iterator))
 }
 
+fn char_to_byte(char_val: Option<char>) -> Result<Option<u8>, DaftError> {
+
+    match u8::try_from(char_val.unwrap()){
+        Err(_e) => Err(DaftError::ValueError(format!(
+            "character is not valid : {:?}",
+            char_val
+        ))),
+        Ok(c) => Ok(Some(c)),
+    }
+}
```

Comment:
Same here, where we could push this down to `daft_csv` and have `daft_csv::read::read_csv()` and `daft_csv::metadata::read_csv_schema()` use it under the hood, and `read_csv_schema()` would instead take `Option<char>` in its API.